ssf/samples.go

```diff
@@ -1,6 +1,15 @@
 package ssf
 
-import "time"
+import (
+	"math/rand"
```

Comment:
`crypto/rand` :)

Reply:
I knew you were going to call that out, and the only defense/justification I have is that the dogstatsd library uses math/rand also, for a `0.0..1.0` RNG that mustn't fail.

Reply:
`crypto/rand` is also guaranteed not to fail or block, and we generally want to be using that unless there's a compelling reason not to. Is there a reason we need deterministic behavior here?

Reply:
soooo `crypto/rand.Int` does return an error, are you saying it's fine to ignore that? If so, I'm sure I can hack a thing together (:

Reply:
Ah! `Sampled` was meant to be called by code constructing the samples, not by `SendSync` or any of the other sending functions - it's a constructor (that removes data, hah), rather than anything else - this is because the ideal use case for those `SSFSample`s is that you'd attach the samples you wish to send to your span directly; once you send the span, it's already too late to toss out samples.

Since this operates as an in-memory calculation, is meant to be composable, and has no expectation of even a chance failure, I think this is a place where determinism would be right.

Reply:
I've made a benchmark for sampling a million metrics at `0.2`; looks fairly ok:

```
pkg: github.com/stripe/veneur/ssf
BenchmarkSampled-4      2000000000               0.03 ns/op
```